core/src/main/java/org/mule/runtime/core/api/context/notification/FlowStackElement.java

```diff
@@ -22,11 +24,17 @@ public final class FlowStackElement implements Serializable {
   private final String flowName;
   private final String processorPath;
   private final long creationTime;
+  private final transient ComponentIdentifier chainIdentifier;
 
   public FlowStackElement(String flowName, String processorPath) {
+    this(flowName, processorPath, null);
+  }
+
+  public FlowStackElement(String flowName, String processorPath, ComponentIdentifier componentIdentifier) {
```

Comment:
leave `processorPath` as the last param of this method, since the other two are related to each other.